spec/features/global_announcements_spec.rb

```diff
@@ -17,9 +17,9 @@
 
       scenario 'I should not see any announcements if there are none' do
         visit announcements_path
-        it { is_expected.not_to have_selector('div.global-announcement') }
-        it { is_expected.not_to have_selector('div.instance-announcement') }
-        it { is_expected.not_to have_selector('div.system-announcement') }
+        expect(page).not_to have_selector('div.global-announcement')
```

Comment:
@lowjoel Just curious, why you prefer `expect(page)` over `it { is_expected }` ?